examples/textual_inversion/textual_inversion.py

```diff
@@ -392,6 +392,7 @@ def main():
 
     # Initialise the newly added placeholder token with the embeddings of the initializer token
     token_embeds = text_encoder.get_input_embeddings().weight.data
+    original_token_embeds = text_encoder.detach().clone()
```

Comment:
Hi!
This line should be:
```python
    original_token_embeds = text_encoder.get_input_embeddings().weight.data.detach().clone()
```
or
```python
    original_token_embeds = token_embeds.detach().clone()
```
As you only need a copy of the embeddings matrix

Reply:
@destevez-kaleido Thank you, that's right. Actually I updated this line in a latter commit. Check it out.

Reply:
You are right, sorry for that. I just realized I had read the commits in the reverse order.